sanicbot/extensions/git.py

```diff
@@ -44,6 +45,26 @@ async def retrieve_github_issue(
             repo = f"sanic-{repo}"
         await self.lookup(ctx, number, repo)
 
+    @commands.command()
+    async def issue(self, ctx: Context, repo: str, title: str, *, content: str) -> None:
+        repo = repo if not repo.startswith("sanic") else "sanic-" + repo
+        url = f"https://api.github.com/repos/sanic-org/{repo}/issues"
+        body = f"Forwarded from discord (Author: {ctx.author}, ID: {ctx.author.id})\n" + content
+        data = {
+            "title": title,
+            "body": body,
+        }
+        headers = {
+            "Accept": "application/vnd.github.v3+json",
+            "Authorization": "token " + config["SANIC"]["git_token"]
+        }
+        async with httpx.AsyncClient() as client:
```

Comment:
by extending the commands.Bot class (see PR #8 for implementation) we can attach a bot-level `httpclient` so we're not creating a new client each time a command is called.